src/components/LangSelector/readme.md

```diff
@@ -2,10 +2,44 @@
 
 Use this component for projects that need have i18n internationalization and need a selector in the header.
 
+## Usage
+
+Selector should be imported within the Phila standards app header. (Component background is clear with white font.)
+
+## Local Storage
+
+Language selector uses local storage to save the current locale for your application.
+
 ## Dependencies
 
 i18n needs to be installed within your project with locales for this component to function. 
-See vue-i18n documentation for more info at http://kazupon.github.io/vue-i18n/
+See vue-i18n documentation for more info at http://kazupon.github.io/vue-i18n/ for the following basic settings:
+
+```js static
+import Vue from 'vue'
+import VueI18n from 'vue-i18n'
```

Comment:
Excellent documentation! 👏 

Can these examples be formatted to match our JS linting rules? Eg: Add commas and semis where appropriate.

Reply:
@karissademi I told @stevetotheizz0 to not worry about the styles right now because the docs are not up yet, so he wouldn't have a way to check. The documentation is also a PR, but I need the local-fonts PR merged first because it has some breaking changes.

Reply:
@ibdf no prob